### DISCUSSION TITLE
Channel took 2 months to fund and is now inactive

### DISCUSSION THREAD

**Discussion by bitsam** [raspiblitz/raspiblitz#4414] [Q&A]
I had an issue with the fee set too low, and then the fees jumped much higher, and the chain transaction took over a month to be confirmed. The chain transaction has now been confirmed and the channel is open at my end, however showing as inactive. I read somewhere if a channel takes over two weeks it is wiped from the other end's database. Is that still the case and do I need to close the channel and open a new one? Is there any other way to get the channel active?

**Comment by rootzoll** [comment]
An inactive channel indicates that it is not currently being used for routing payments. This can happen if the node on the other end of the channel is offline or if there's an issue with the channel's state. So first check on that and with the counter party of the channel if possible.

About channels being wiped from the other node's database if a channel takes over two weeks to become active, that would be best answered directly with the support channels of LND or CLN, because this can vary from implementaion & version and they need the feedback to better improve their user-experience.
